.github/workflows/build-and-publish-rule-book.yaml

```diff
@@ -94,30 +111,25 @@ jobs:
 
       # Printable build goes below
       - uses: actions/setup-python@v5
-        if: github.event_name != 'pull_request'
         with:
           python-version: "3.11"
 
       - name: Add pages to select hyperlinks
-        if: github.event_name != 'pull_request'
         run: |
           find ${{ matrix.sections }} -type f -execdir sed -i 's@\\hypertarget@\\pagetarget@g' '{}' +
           python .github/insert_printable_hyperlinks.py ${{ matrix.sections }}
 
       - name: Include notes page
-        if: ${{ github.event_name != 'pull_request' && env.EVEN_NUMBER_OF_PAGES == 'true' }}
+        if: ${{ env.EVEN_NUMBER_OF_PAGES == 'true' }}
         run: sed -i 's@% Notes placeholder@\\include{\\sections/notes.tex}@g' metadata.tex
 
       - name: Include index page
-        if: github.event_name != 'pull_request'
         run: sed -i 's@\\include{\\sections/back_cover.tex}@\\include{\\sections/index.tex}\\include{\\sections/back_cover.tex}@g' metadata.tex
 
       - name: Add QR codes
-        if: github.event_name != 'pull_request'
         run: sed -i -e '/% QR codes placeholder/{r .github/qr-codes-${{ matrix.language }}.tex' -e 'd}' metadata.tex
 
       - name: Compile printable rule book
-        if: github.event_name != 'pull_request'
```

Comment:
Looks like this will make the checks longer run longer for PRs

Reply:
Yes, no way around it.